modules/login/php/requestaccount.class.inc

```diff
@@ -271,5 +271,53 @@ class RequestAccount extends \NDB_Form
 
         // Show success message even if email already exists for security reasons
         $this->tpl_data['success'] = true;
+
+        // Don't send out emails if this is a sandbox VM.
+        if ($factory->config()->settingEnabled('sandbox')) {
+            return;
+        }
+
+        // Send an email to all users meeting the following criteria:
+        //     - Associated with the Site
+        //     - Associated with the Project
+        //     - Have the User Management permission
+        $query  = <<<SQL
```

Comment:
This still needs to be changed from a HEREDOC to normal PHP quotes by whoever picks this up, then it'll be ready to be merged.